app/components/AddThis.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+interface Props {
+  url: string;
+  title: string;
+}
+
+declare const window: any;
+
+export default class Card extends React.Component<Props, {}> {
+  componentDidMount() {
+    window.addEventListener('load', window.addthis.layers.refresh());
+  }
+
+  componentDidUpdate() {
+    window.addEventListener('load', window.addthis.layers.refresh());
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('load', window.addthis.layers.refresh());
+  }
+
+  render() {
+    const card_url = `https://www.aliemcards.com${this.props.url}`;
```

Comment:
This is potentially dangerous here. XSS injection, etc.

Need to probably sanitize it or something to make sure its a good url without any query parameters or fluff.

Reply:
This is just the pathname passed down from React Router. No user input or url parameters used.